src/components/projects/projects-table/projects-table/projects-table.tsx

```diff
@@ -34,11 +34,52 @@ const ProjectsTable: React.FC<DisplayProjectProps> = ({ allProjects }: DisplayPr
   // TODO: Sort by wbsNum means 1.1.0 > 1.12.0 > 1.2.0, but desired is 1.1.0 > 1.2.0 > 1.12.0
   // TODO: Sort by duration means 12 > 2 > 4 > 5 > 9, but desired is 12 > 9 > 5 > 4 > 2
   const columns: ColumnDescription[] = [
-    { dataField: 'wbsNum', text: 'WBS #', align: 'center', sort: true },
-    { dataField: 'name', text: 'Name', align: 'left', sort: true },
-    { dataField: 'projectLead', text: 'Project Lead', align: 'left', sort: true },
-    { dataField: 'projectManager', text: 'Project Manager', align: 'left', sort: true },
-    { dataField: 'duration', text: 'Duration', align: 'center', sort: true }
+    {
+      headerAlign: 'center',
+      dataField: 'wbsNum',
+      text: 'WBS #',
+      align: 'center',
+      sort: true,
+      sortFunc: (a, b, order: SortOrder) => {
+        const a1 = a.split('.');
+        const b1 = b.split('.');
+        const len = Math.min(a1.length, b1.length);
+        for (let i = 0; i < len; i++) {
+          const a2 = +a1[i] || 0;
```

Comment:
avoid using unintelligible variable names. it quickly becomes difficult to trace which variable is what